### PR TITLE
Clarify that China support is community run

### PR DESCRIPTION
**Pull request by mit-mit**

### CODE REVIEW

**Comment by chalin** [comment]
@mit-mit - I did a copyediting pass, and pushed an extra commit atop of yours. PTAL

The changes are staged: https://flutter-io-staging-2.firebaseapp.com/community/china

**Comment by mit-mit** [comment]
The yellow box applies to the whole page, not just the last mirrors, so your change to move it won't work, I think

**Comment by chalin** [comment]
> The yellow box applies to the whole page

Got it (I wasn't sure). In that case, the warning should appear at the top of the page.
Any objections? If not, I'll move it there.

**Comment by chalin** [comment]
I've made the update and refreshed the staging server. Let me know what you think: https://flutter-io-staging-2.firebaseapp.com/community/china

File: src/community/china.md [hunk 1]
```diff
@@ -12,16 +13,20 @@ you can replace the domain of the original URL with a trusted mirror
...
 
 You  still need to set up two environment variables to upgrade Flutter
-and use the pub package repository in China.
-Please follow the instructions below.
+and use the pub package repository in China. Instructions are given below.
 
----
+{{site.alert.important}}
+  Use mirror sites only if you _trust_ the provider.
+  The Flutter team cannot verify their reliability nor security.
```
Comment (kwalrath) [inline src/community/china.md]: nor -> or

(Not sure "nor" is wrong here, but we don't use it much this way.)

File: src/community/china.md [hunk 2]
```diff
@@ -12,16 +13,20 @@ you can replace the domain of the original URL with a trusted mirror
...
 
 * Original URL:<br>
-  [https://storage.googleapis.com/flutter_infra/releases/beta/windows/flutter_windows_v0.3.2-beta.zip](https://storage.googleapis.com/flutter_infra/releases/beta/windows/flutter_windows_v0.3.2-beta.zip)
+  [https://storage.googleapis.com/flutter_infra/releases/stable/windows/flutter_windows_v1.0.0-stable.zip](https://storage.googleapis.com/flutter_infra/releases/stable/windows/flutter_windows_v1.0.0-stable.zip)
 
 * Mirrored URL:<br>
-  [https://storage.flutter-io.cn/flutter_infra/releases/beta/windows/flutter_windows_v0.3.2-beta.zip](https://storage.flutter-io.cn/flutter_infra/releases/beta/windows/flutter_windows_v0.3.2-beta.zip)
+  [https://storage.flutter-io.cn/flutter_infra/releases/stable/windows/flutter_windows_v1.0.0-stable.zip](https://storage.flutter-io.cn/flutter_infra/releases/stable/windows/flutter_windows_v1.0.0-stable.zip)
 
 You  still need to set up two environment variables to upgrade Flutter
```
Comment (kwalrath) [inline src/community/china.md]: still need to set up -> must also set

File: src/community/china.md [hunk 3]
```diff
@@ -12,16 +13,20 @@ you can replace the domain of the original URL with a trusted mirror
...
-and use the pub package repository in China.
-Please follow the instructions below.
+and use the pub package repository in China. Instructions are given below.
```
Comment (kwalrath) [inline src/community/china.md]: Delete "given".

**Comment by chalin** [comment]
@kwalrath - I made a few more changes in formatting PTAL. Also, could you fix the CLA?

**Pull request merged**